### DISCUSSION TITLE
希望提供多批次添加相同label的点和边的功能。

### DISCUSSION THREAD

**Discussion by vipkelvinli** [alibaba/GraphScope#1094] [General]
**Is your feature request related to a problem? Please describe.**
我们想把graphscope用在线上系统，应用场景如下：
1:先将离线图数据批量加载进vineyard内存中。
2:通过消息队列实时接收在线业务订单数据。
3:结合订单数据和离线数据进行实时构图。
4:图的顶点和边的label不止一种类型。

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sighingnow** [comment]
有几个具体的问题哈，

1. 这个每次变更之后的图，是用来run哪种类型的query？是analytical的还是interactive query？
2. 每次新增的数据量和离线图的数据量的比例大概是怎样的？

**Comment by vipkelvinli** [comment]
> 有几个具体的问题哈，
> 
> 1. 这个每次变更之后的图，是用来run哪种类型的query？是analytical的还是interactive query？
> 2. 每次新增的数据量和离线图的数据量的比例大概是怎样的？

问题1:每次变更后是interactive query，小范围的子图查询。
问题2:每次新增数量在100以内，离线图数据在百万级别。

**Comment by sighingnow** [comment]
@siyuan0322 @tianliplus  帮忙介绍一下怎么用graphscope-store + gaia处理这个case，包括怎么拉起，用户怎么load图，怎么update，怎么query这些。现在这部分有类似tutorial的文档吗？

**Comment by siyuan0322** [comment]
> @siyuan0322 @tianliplus 帮忙介绍一下怎么用graphscope-store + gaia处理这个case，包括怎么拉起，用户怎么load图，怎么update，怎么query这些。现在这部分有类似tutorial的文档吗？

看起来可以通过先Bulk load 图数据然后通过实时写更新数据来做。参考一下这个文档  https://graphscope.io/docs/persistent_graph_store.html
